src/api/hooks/useGetDelegationNodeInfo.ts

```diff
@@ -21,27 +30,46 @@ export function useGetDelegationNodeInfo({
   validator,
 }: DelegationNodeInfoProps): DelegationNodeInfoResponse {
   const [state, _] = useGlobalState();
-  const [commission, setCommission] = useState<Types.MoveValue>();
-  const client = new AptosClient(state.network_value);
   const {totalVotingPower} = useGetValidatorSet();
+  const {data, isLoading} = useGetAccountResources(validatorAddress);
```

Comment:
looks like you are looking for a specific resource - can use [useGetAccountResource](https://github.com/aptos-labs/explorer/blob/main/src/api/hooks/useGetAccountResource.ts)